downstream/modules/platform/con-edge-manager-provisioning-openshift-virt.adoc

```diff
@@ -0,0 +1,12 @@
+[id="edge-manager-provisioning-openshift-virt"]
+
+= Provision devices on OpenShift Virtualization
```

Comment:
Is "on" the correct preposition here? Could you say "Provision devices with OpenShift Virtualization"
As in, are you provisioning on openshift using openshift virtualization?
I could be wrong here!

Also, I think it might be useful to create an attribute for OpenShift Virtualization